src/arrayMethodSort.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
-/**
+/*
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = undefined) {
+    const myOwnCompare =
+      compareFunction || ((a, b) => (a.toString() > b.toString() ? 1 : -1));
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = defaultCompareFunction) {
```
- You can move outside the function and as a default parameter
- Name is strange